docs/platforms.md

```diff
@@ -0,0 +1,88 @@
+# Using InSpec 2.0 with Platforms
+
+New in InSpec 2.0, you may now use certain InSpec resources to audit properties of things that aren't individual machines - for example, an Amazon Web Services S3 bucket.
+
+In the initial release of InSpec 2.0, support for selected AWS and Azure resources is included.
+
+## AWS Platform Support in InSpec 2.0
+
+### Setting up AWS credentials for InSpec
+
+InSpec uses the standard AWS authentication mechanisms.  Typically, you will create an IAM user which will be used for auditing activities.
+
+1. Create an IAM user in the AWS console, with your choice of username.  Check the box marked "Programmatic Access."
+2. On the Permissions screen, choose Direct Attach.  Select the AWS-managed IAM Profile named "ReadOnlyAccess."  If you wish to restrict the user further, you may do so; see individual InSpec resources to identify which permissions are required.
+3. After the key is generated, record the Access Key ID and Secret Key.
+
+#### Using Environment Variables to provide credentials
+
+You may provide the credentials to InSpec by setting the following environment variables: `AWS_REGION`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_KEY_ID`.  You may also use `AWS_PROFILE`, or if you are using MFA, `AWS_SESSION_TOKEN`.  See the [AWS Command Line Interface Docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) for details.
+
+#### Using the InSpec target option to provide credentials
+
+Look for a file in your home directory named `~/.aws/credentials`.  If it does not exist, create it.  Add your credentials as a new profile, in INI format:
+
+```
+[my-profile-name]
+aws_access_key_id = AKIA....
+aws_secret_access_key = 1234....abcd
+```
+
+You may now run InSpec using the `--target` / `-t` option, using the format `-t aws://region/profile`.  For example, to connect to the Ohio region using a profile named 'auditing', use `-t aws://us-east-2/auditing`.
+
```

Comment:
Should we include the `inspec detect` as an example to verify that the credentials and the connection works?

Reply:
This may be misleading.  Train's AWS support doesn't test the connection until an API call is actually tried.